lib/generators/curation_concerns/work/templates/form_spec.rb.erb

```diff
@@ -0,0 +1,10 @@
+# Generated via
+#  `rails generate curation_concerns:work <%= class_name %>`
+require 'rails_helper'
+
+describe CurationConcerns::<%= class_name %>Form do
+  it "should have some tests" do
```

Comment:
Could we `s/should have/has/` here to encourage writing tests that conform to our style?